### PR TITLE
feat(blue-sdk): add wagmi-like hooks

### PR DESCRIPTION
**Pull request by Rubilmax**

### CODE REVIEW

File: packages/blue-sdk-wagmi/src/hooks/useChainId.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export type UseChainIdParameters<config extends Config = Config> =
+  ChainIdParameter<config> & ConfigParameter<config>;
+
+export function useChainId<config extends Config = ResolvedRegister["config"]>(
+  parameters: UseChainIdParameters<config>,
+) {
+  const wagmiChainId = wagmi_useChainId(parameters);
+
+  return ChainUtils.parseSupportedChainId(parameters.chainId ?? wagmiChainId);
```
Comment (oumar-fall) [inline packages/blue-sdk-wagmi/src/hooks/useChainId.ts:18]: Is it convenient to have react hooks throw errors?

**Review by oumar-fall — APPROVED** [review]
looks promising! I think some comments might arise with usage

File: packages/blue-sdk-wagmi/src/hooks/useChainId.ts (hunk 1 above)
Comment (Rubilmax) [reply packages/blue-sdk-wagmi/src/hooks/useChainId.ts:18]: Nice catch, I'll try to find a workaround

Comment (Rubilmax) [reply packages/blue-sdk-wagmi/src/hooks/useChainId.ts:18]: I actually think it's fine to break everything if wagmi is connected to a chain id not accepted by the sdk

**Pull request merged**